### DISCUSSION TITLE
In-game leaderboard rank should not show #51

### DISCUSSION THREAD

**Discussion by reo-the-enby** [ppy/osu#20538] [Ideas] [answered]
I think the in-game leaderboard should not show #51 when a map has more than 50 scores on the set leaderboard.
Say you have global ranks on, you wouldn't actually be #51 in the beginning. I think the rank should only display if you pass the score at #50 since then you will be that rank after the map is finished.

<img width="197" alt="Skärmavbild 2022-09-30 kl  12 23 09" src="https://user-images.githubusercontent.com/81715029/193249981-99026acb-f2f4-453e-8346-e3bc2ab8e9af.png">

**Comment by peppy** [comment, marked as answer]
Honestly I think I'm going to leave this as it is for now. Going forward we have hopes to display far more than the top 50 scores, at which point this will become less relevant.

While it's showing 51 scores, they *are* correct. So you're just getting one bonus score for now.
